aiohue/v1/groups.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, logger: Logger, raw: Dict[str, Any], request: Coroutine) -> N
 
     async def get_all_lights_group(self) -> "Group":
         """Return special all lights group."""
-        return Group("0", await self._request("get", "groups/0"), [], self._request)
+        return Group("0", await self._request("get", "groups/0"), self._request)
```

Comment:
Is this a bug fix? It looks like the `Group` signature didn't match the call here before.

Reply:
Exactly, I discover that by accident

Reply:
Is Home Assistant affected, and if so, is everything else backwards compatible for library users so we can release this in a patch release of Home Assistant, without having to update anything else?

Reply:
Yes, no problem.